app/views/organization_account_requests/new.html.erb

```diff
@@ -0,0 +1,106 @@
+<section class="pt-5 pb-5">
+  <div class="container mb-md-5 mt-md-5 ">
+    <div class="container mb-md-5 mt-md-5">
+      <div class="text-center mb-5">
+        <h1 class="section-heading text-capitalize underline">
+          <%= t('.header') %>
+        </h1>
+      </div>
+    </div>
+
+    <div class='mb-3 text-center bigger'>
+      <%= t('.description') %>
+    </div>
+    <div class="row">
+      <div class="col-md-6 mx-auto card p-5">
+      <%= bootstrap_form_with model: Organization, :url => organization_account_requests_path do |form| %>
+        <% if @organization_acccount_request.errors.count > 0 %>
+          <div class="alert alert-danger mt-1" role="alert">
+            <%= t('organizations.staff.organizations.form.please_fix_the_errors') %>
+          </div>
+          <ul>
+            <% @organization_acccount_request.errors.full_messages.each do |message| %>
+              <div class="alert alert-danger mt-1" role="alert">
+                <%= message %>
+              </div>
+            <% end %>
+          </ul>
+        <% end %>
+
+        <div class='card-body'>
+          <div class="row mt-3">
+            <div class="col-lg-6" data-controller="slug-from-title">
+              <form class="mb-3">
+
+                <div class="form-group">
+                  <%= form.text_field :name,
+                    label: 'Organization name',
+                    class: 'form-control',
+                    data: {
+                      slug_from_title_target: 'title',
+                      action: 'keyup->slug-from-title#updateSlug'
+                    } %>
+                </div>
```

Comment:
Probably organization name and email should also be required. Possibly also phone number?